.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/usr/bin/python2"
```

Comment:
Looks like you accidentally added this file.

Reply:
just a sec, will fix that